docs/dev-guide/getting-started.md

```diff
@@ -1,11 +1,49 @@
 # Developer Guide - Getting Started
 
-Getting started text. Also, add additional pages as needed.
+* Prereqs
+    * Java 8
+    * Maven 3.x
+    * Git Tools
+    * Eclipse _Optional_
+
+## Prereqs
+
+### Java 8 (or above) 
+
+The quality measure and cohort engine has been tested with OpenJDK8 and OpenJDK11. To install Java on your system, we recommend downloading and installing Java 11 OpenJ9 from [https://adoptopenjdk.net](https://adoptopenjdk.net)
+
+### Maven 3.x (or above)
+
+The quality measure and cohort engine is a multi-module Maven project. Any recent version of Maven should work. The team uses 3.6.3. Download from the [Maven download site](https://maven.apache.org/download.cgi).
+
+### Git Tools
+
+To download the repository or a fork, you should have Git Tools installed. Download and install from [git-scm](https://git-scm.com/downloads).
+
+### Eclipse
+
+Optional: If you want or do use Eclipse.
+
+Download [Eclipse IDE for Enterprise Java Developers](https://www.eclipse.org/downloads/packages/release/2020-06/r/eclipse-ide-enterprise-java-developers) and install the following plugins:
+
+## Build the code
+
+Git fork and clone the quality measure and cohort engine repository using GitHub Standard Fork & Pull Request Workflow. For instance, ``git clone git@github.com:<myuser>/FHIR.git`` where ``myuser`` is your GitHub user or GitHub organization.
```

Comment:
Is it worth it to swap out FHIR with our repo name?

Reply:
I will fix the formatting, remove the reference to plugins, and fix the copy/paste error in the repo name.